src/bidiMapper/domains/network/NetworkProcessor.ts

```diff
@@ -163,8 +163,37 @@ export class NetworkProcessor {
     return {};
   }
 
-  provideResponse(_params: Network.ProvideResponseParameters): EmptyResult {
-    throw new UnknownCommandException('Not implemented yet.');
+  async provideResponse(
+    params: Network.ProvideResponseParameters
+  ): Promise<EmptyResult> {
+    const networkId = params.request;
+    const {request: fetchId} = this.#getBlockedRequest(networkId);
+
+    const {statusCode, reasonPhrase, headers, body} = params;
+
+    // TODO: Step 6
+    // https://w3c.github.io/webdriver-bidi/#command-network-continueResponse
+
+    const responseHeaders: Protocol.Fetch.HeaderEntry[] | undefined =
+      cdpFetchHeadersFromBidiNetworkHeaders(headers);
+
+    // TODO: Set / expand.
+    // ; Step 10. cookies
+    // ; Step 11. credentials
+    const request = this.#networkStorage.getRequest(networkId);
+    assert(request, `Network request with ID ${networkId} doesn't exist`);
+
+    await request.provideResponse(
+      fetchId,
+      statusCode!, // TODO: What if the status code is not present?
```

Comment:
Provide one from the request, it should be there.

Reply:
We still need to provide a fallback in case it is undefined. I am using 200. Is that OK?